packages/file_selector/file_selector/example/lib/open_multiple_images_page.dart

```diff
@@ -0,0 +1,85 @@
+import 'dart:io';
+import 'package:flutter/foundation.dart';
+import 'package:file_selector/file_selector.dart';
+import 'package:flutter/material.dart';
+
+/// Screen that shows an example of openFile(s)
+class OpenMultipleImagesPage extends StatelessWidget {
+  void _openImageFile(BuildContext context) async {
+    final XTypeGroup jpgsTypeGroup = XTypeGroup(
+      label: 'images',
```

Comment:
It seems odd to call this "images" rather than "JPEGs" given that the other option is also an image type.

Reply:
Done!